.github/workflows/ci.yml

```
@@ -22,7 +22,7 @@ jobs:
         - name: Download dependencies
           run: |
             git clone -b ros2 https://github.com/eProsima/Micro-CDR src/Micro-CDR
-            git clone -b ros2 https://github.com/eProsima/Micro-XRCE-DDS-Client src/Micro-XRCE-DDS-Client
+            git clone -b feature/update_bin_api https://github.com/eProsima/Micro-XRCE-DDS-Client src/Micro-XRCE-DDS-Client
```

```suggestion
            git clone -b fros2 https://github.com/eProsima/Micro-XRCE-DDS-Client src/Micro-XRCE-DDS-Client
```

When https://github.com/eProsima/Micro-XRCE-DDS-Client/pull/294 merged